client/src/components/relatedItems/RelatedItems.jsx

```diff
@@ -7,68 +7,24 @@ import withTracker from '../../components/Sharables/withTracker.js';
 var RelatedItems = (props) => {
   const {
     productId,
+    relatedProduct,
     yourOutfit,
     onCardClick,
     onStarClick,
     onXClick
   } = props;
 
-  const [productInfo, setProductInfo] = useState({});
-  const [productStyle, setProductStyle] = useState({});
-  const [relatedProduct, setRelatedProduct] = useState([]);
-  const [relatedProductInfo, setRelatedProductInfo] = useState([]);
-  const [relatedProductReviews, setRelatedProductReviews] = useState([]);
-  const [relatedProductStyles, setRelatedProductStyles] = useState([]);
-  const [yourOutfitInfo, setYourOutfitInfo] = useState([]);
-  const [yourOutfitReviews, setYourOutfitReviews] = useState([]);
-  const [yourOutfitStyles, setYourOutfitStyles] = useState([]);
+  const [productInfo, setProductInfo] = useState([]);
+  const [productReviews, setProductReviews] = useState([]);
+  const [productStyles, setProductStyles] = useState([]);
 
-  const getProductInfo = ((productId, listType) => {
-    let productInfoRequest = axios.get(`/products/${productId}`);
-    let productStyleRequest = axios.get(`/products/${productId}/styles`);
-    let relatedProductRequest = axios.get(`/products/${productId}/related`);
+  let allProduct = Array.from(new Set([productId, ...relatedProduct, ...yourOutfit])).sort();
```

Comment:
Not too sure what this line does, but my guess is you're consolidating all productId, relatedProduct and yourOutfit into 1 array so the useEffect can check on it?